src/System.Management.Automation/help/UpdatableHelpCommandBase.cs

```diff
@@ -514,7 +514,6 @@ private void ProcessModule(UpdatableHelpModuleInfo module)
                     ProcessException(module.ModuleName, culture, new UpdatableHelpSystemException("AccessIsDenied",
                         e.Message, ErrorCategory.PermissionDenied, null, e));
                 }
-#if !CORECLR
```

Comment:
Changes in this file need to be careful. The implementation was changed from using `WebClient` to `HttpClient`, and thus some `'if !CoreCLR'` blocks don't really make sense anymore. Let's leave this file aside for now.